assets/scss/_custom.scss

```
@@ -125,6 +125,33 @@ body.td-home main[role="main"] > section:first-of-type .content p:first-child {
   }
 }
 
+.section-feature#kubeweekly {
+
+  a.kubeweekly-signup, a.kubeweekly-signup:hover {
+    color: $primary
```

```suggestion
    color: $primary;
```
You forgot a semicolon 😛